### PR TITLE
Stabilize jaeger docker tests

### PR DESCRIPTION
**Pull request by juangon**
Have been executing using this successfully in our environment.

### CODE REVIEW

File: common/src/main/java/org/wildfly/swarm/ts/common/docker/Docker.java [hunk 1]
```diff
@@ -21,6 +21,7 @@
     private final String image;
     private final List<String> ports = new ArrayList<>();
     private final Map<String, String> environmentVariables = new HashMap<>();
+    private final Map<String, String> variables = new HashMap<>();
```
Comment (Ladicek) [inline common/src/main/java/org/wildfly/swarm/ts/common/docker/Docker.java]: So, these are not variables, these are command arguments. `var` doesn't say anything.

**Comment by juangon** [comment]
@Ladicek pushed your suggestion. BTW tests seems pretty stable now.

**Comment by Ladicek** [comment]
Great to hear. Now that I'm thinking about it again, not all containers will accept arguments in form of `--<key>=<value>`, so I'm going to refactor this a little.

**Comment by Ladicek** [comment]
Thanks!

**Pull request merged**